config/routes.rb

```diff
@@ -5,5 +5,7 @@
   resources :turns
   resources :groups
 
-  devise_for :users
+  unless ENV['PRECOMPILE']
```

Comment:
Devise requires ActiveRecord, which tries to load the database configuration in production